calico_kubernetes/calico_kubernetes.py

```diff
@@ -85,7 +90,25 @@ def delete(self, namespace, pod_name, docker_id):
         try:
             self._datastore_client.remove_profile(self.profile_name)
         except:
-            print "Cannot remove profile %s; Profile cannot be found." % self.profile_name
+            logger.warning("Cannot remove profile %s; Profile cannot be found.",
+                           self.profile_name)
+
+    def status(self, namespace, pod_name, docker_id):
```

Comment:
As discussed before, could you keep the logging / status hook / IPAM code in separate pull requests?

The logging shouldn't be dependent on the status code being right.